### DISCUSSION TITLE
How to wrap / auto-break long lines?

### DISCUSSION THREAD

**Discussion by ThomasGrafNumerics** [NvChad/NvChad#2545] [Q&A]
For C++ I use the clang-formater that allows me to set a maximum text width but how do I make NvChad break / wrap long lines of text for example in a .txt-file?
I have tried things like putting 'vim.opt.breakline = 120' into my custom init.lua but it seems that something is overwritting this option. How do you handle this. I'm assuming a fresh / unaltered NvChad configuration.
![issue](https://github.com/NvChad/NvChad/assets/76207883/fb86ebb8-b3ac-47c6-aa78-71f7c8ac3302)
Thank you so much!

**Comment by siduck** [comment]
@ThomasGrafNumerics comment your code and reopen nvim, print the value of breakline opt , if its there then probably some plugin's doing it

**Reply by ThomasGrafNumerics** [reply]
Highly appreciated answer. I noticed that I had to press 'gqG' do make the formatting to the set textwidth happen. Thanks again!
